code/modules/atmospherics/machinery/components/unary_devices/vent_scrubber.dm

```diff
@@ -86,21 +86,6 @@
 	. = ..()
 	disconnect_from_area(area_to_unregister)
 
-///adds a gas or list of gases to our filter_types. used so that the scrubber can check if its supposed to be processing after each change
```

Comment:
It's unused and because I'm adding ``update_filters`` for monkeys to use, it considered it a double-define, this is from the PR I'm porting https://github.com/tgstation/tgstation/pull/81598